tests/unit/other.tcl

```diff
@@ -350,29 +356,6 @@ start_server {tags {"other"}} {
 }
 
 start_server {tags {"other external:skip"}} {
-    test {MEMORY PURGE basic} {
-        if {[string match {*jemalloc*} [s mem_allocator]]} {
-            r debug populate 70000 asdf1 1500
-            set mem [s used_memory_rss]
-            r flushdb async
-
-            # Wait until there are some unused memory pages.
-            wait_for_condition 50 100 {
-                [s used_memory] + (4 * 1024 * 1024) < $mem
-            } else {
-                fail "Memory is not reclaimed by FLUSHDB ASYNC"
-            }
-
-            r memory purge
-
-            wait_for_condition 50 100 {
-                [s used_memory_rss] < $mem
-            } else {
-                fail "Memory is not reclaimed by MEMORY PURGE"
-            }
-        }
-    }
```

Comment:
did we write a better test and decided to drop it?

Reply:
this test does a bit more than coverage and was flaky (I've seen it fail on "Memory is not reclaimed by MEMORY PURGE")

since this PR is big enough already, i don't want to add tests with "meat"

Reply:
(it's a new test introduced in one of the commits of this PR, it doesn't exist in unstable)